src/components/Toggle/__stories__/toggle.stories.mdx

```diff
@@ -42,12 +41,12 @@ Allow users to turn an single option on or off. They are usually used to activat
 <UsageGuidelines guidelines={[
   "Use toggles when your intent is to turn something on or off instantly.",
   "Let users know what happens when the toggle is switched by using a tooltip.",
-  "Toggle can either be selected or not selected. They cannot be in an indeterminate state (unlike checkboxes).",
+  <>Toggle can either be selected or not selected. They cannot be in an indeterminate state (unlike <Link href="/?path=/docs/components-checkboxes--overview">checkboxes</Link>).</>,
```

Comment:
change the link to href="/?path=/docs/components-checkbox--overview

the current link is broken